tests/integration/scenarios/tls/tests/test_tls_sni_001.py

```
@@ -0,0 +1,140 @@
+                receiver.requests.append(
+                    {
+                        "path": self.path,
+                        "headers": dict(self.headers),
+                        "body": body,
+                    }
+                )
+                self.send_response(200)
+                self.end_headers()
+                self.wfile.write(b"ok")
+
+            def log_message(self, format, *args):
```

Ruff (0.15.13)

[error] Function argument `format` is shadowing a Python builtin

(A002)